recipe/meta.yaml

```diff
@@ -27,6 +25,8 @@ build:
     - openssl  # [not win]
     - zlib
     - abseil-cpp  # [win]
+    - brotli
+    - libbrotlicommon
```

Comment:
These shouldn't be here. It suggests that either the brotli feature was not actually enabled or brotli got statically linked. We should address those problems directly, rather than simply inserting `ignore_run_exports` entires.